### PR TITLE
Enhance gRPC errors

### PR DESCRIPTION
**Pull request by matiasinsaurralde**
Enhancement for #991 
The dialer already does an URL check (parsing) but this is a simpler thing to check during init

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -190,7 +190,11 @@ func setupGlobals() {
 	}
 
 	if config.Global.CoProcessOptions.EnableCoProcess {
-		CoProcessInit()
+		if err := CoProcessInit(); err != nil {
+			log.WithFields(logrus.Fields{
```
Comment (asoorm) [inline main.go]: Can just use `log.WithField("prefix", "coprocess")`

Comment (matiasinsaurralde) [reply main.go]: Updated

**Comment by matiasinsaurralde** [comment]
This is a very small change, to test this, set this into your `tyk.conf` and use the standard Tyk build (the one we get from `go build -tags 'coprocess grpc'`:
```json
     "coprocess_options": {
        "enable_coprocess": true,
        "coprocess_grpc_server": ""
    },
```
Without the patch you should see the recurring dialer errors.

**Comment by matiasinsaurralde** [comment]
Just fixed conflicts on this PR

**Pull request merged**